### PR TITLE
janitoring: Fixed grammar.

### PR DESCRIPTION
**Pull request by JakubNvk**
Signed-off-by: Jakub Novak jakubn@imterra.com

### CODE REVIEW

File: projects.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ distinction between repository and project arises, official names are:
 recommended to use one word names only. Get creative!
 
 Each project has its team of developers, who are responsible for the project's
-development, maintaining and monitoring its services, etc.
+development, maintaining and monitoring of its services, etc.
 
-Of course anyone can contrib to any project, but if you are a part of some
-project, you probably should devote most of your time to it.
+Of course anyone can contribute to any project, but if you are a part of some
+other project, you should devote most of your time to it.
```
Comment (pepol) [inline projects.md]: We don't want "other" here. It's meant in the way that you can commit to anything you want but if you have a project, you should focus mostly on that.

**Comment by pepol** [comment]
@JakubNvk looks great, although please address my comment.

File: projects.md (hunk 1 above)
Comment (JakubNvk) [inline projects.md]: Allright.

**Comment by pepol** [comment]
@JakubNvk LGTM! Please squash it so we can merge it.

**Pull request merged**